### ISSUE TITLE
Log in with Unsplash not working

### ISSUE THREAD

**Issue by satellouk** [studioespresso/craft3-unsplash#8]
would like to add my own photos from unsplash but when clicking on the button

Log in with Unsplash

on the plugin page, it goes to a dead page on your website.

Page Not Found
Template not found: nlwp-splashing

**Comment by janhenckens** [comment]
Hey
Are you using the Craft or the WordPress version of this plugin?

Thanks

**Comment by gualtierogrey** [comment]
thanks, it's the wordpress installed directly from WP 

WP 4.9.8
theme Mezmerise 1.6.72

**Comment by gualtierogrey** [comment]
hi again, don't know if anything has been done on the WP plugin, but now it seems to redirect to the Unslpash login page for authorization, but after filling email and pass for my account this error is thrown when i hit AUTHORIZE:

Warning: count(): Parameter must be an array or an object that implements Countable in /data/sites/web/studioespressoco/www/wp-splashing/vendor/guzzlehttp/guzzle/src/Handler/CurlFactory.php on line 67

Warning: Cannot modify header information - headers already sent by (output started at /data/sites/web/studioespressoco/www/wp-splashing/vendor/guzzlehttp/guzzle/src/Handler/CurlFactory.php:67) in /data/sites/web/studioespressoco/www/wp-splashing/index.php on line 52

and next there will be a FORBIDDEN page:

Forbidden
You don't have permission to access /wp-splashing/&session=YTozOntzOjg6InJlZGlyZWN0IjtOO3M6MTE6Im9hdXRoMnN0YXRlIjtzOjMyOiJPcFFBSTBpbjd5N3lMZUppOGdMenhxUXVJRVZRS1VNMiI7czo1OiJ0b2tlbiI7TzozODoiTGVhZ3VlXE9BdXRoMlxDbGllbnRcVG9rZW5cQWNjZXNzVG9rZW4iOjU6e3M6MTQ6IgAqAGFjY2Vzc1Rva2VuIjtzOjY0OiJkNjNmZjg3NjQ4MTUxZDA5MWVhZjU0NWM0MWQ0ZmEwZTBhZjk5ODdlNWYwODkyMWFlYzA2ZjM0NDUxYzM2N2FkIjtzOjEwOiIAKgBleHBpcmVzIjtOO3M6MTU6IgAqAHJlZnJlc2hUb2tlbiI7czo2NDoiYmI0MWYzNzlmMDBkYmNhOTRhMjk0NmRiZjIyYzgzMzFmYzNiMjUyNmUzMWQ2ZGE4MDA5YTZmMWYyYTcyY2ZmZiI7czoxODoiACoAcmVzb3VyY2VPd25lcklkIjtOO3M6OToiACoAdmFsdWVzIjthOjM6e3M6MTA6InRva2VuX3R5cGUiO3M6NjoiYmVhcmVyIjtzOjU6InNjb3BlIjtzOjE2OiJyZWFkX3VzZXIgcHVibGljIjtzOjEwOiJjcmVhdGVkX2F0IjtpOjE1NDAxMTYyNjg7fX19 on this server.

as result on my website > media > splashing images when i click on the 

Log in with Unsplash

the next page shows empty. Hope this helps. If you need logs, please let me know where to find them.

On Unspash applications the plugin shows as authorized.

Regards.

**Comment by gualtierogrey** [comment]
hi, i've just tried again and plugin still got some kind of error.

now it shows 


Page Not Found

Template not found: wp-splashing

after auth. page.

**Comment by janhenckens** [comment]
Yes, working on it! I fixed the server/authentication stuff over the weekend and am releasing the plugin update tonight, that should fix it!
